src/test/java/org/folio/rest/api/OaiPmhViewTest.java

```diff
@@ -385,6 +402,74 @@ public void testResponseStatus400WhenRequestingWithInvalidUntilDate()
     });
   }
 
+  @Test
+  public void test500(TestContext testContext) {
```

Comment:
Under what circumstances should a 500 response be returned?

From the investigation I've done, this is triggered by a null pointer exception thrown because of the null parameter. Was that intentional?

Reply:
500 should be returned when AbstractInstanceRecordsAPI.respondWithError is called, this PR doesn't change this.

Reply:
The null pointer exception is one way to trigger a respondWithError call and is intentional to unit test the catch clause.